changes/5713.feature

```diff
@@ -1,5 +1,10 @@
 Update resources faster by selectively skipping resource validation and
-database updating for unchanged resources.
+database updating for unchanged resources. Now activities are created and
+metadata_modified updated only if there is a real change. metadata_modified
+may now be set by sysadmins (useful for harvesting or mirroring).
+
+The allow_partial_update context parameter has been removed, now normal API
+users may call package_update without passing resources.
```

Comment:
What will be the effect of this?
```suggestion
users may call package_update without passing resources. In this case the existing resources
will remain untouched instead of being deleted.
```